packages/cli-kit/src/node/dot-env.ts

```diff
@@ -50,3 +50,42 @@ export async function readAndParseDotEnv(path: string): Promise<DotEnvFile> {
 export async function writeDotEnv(file: DotEnvFile) {
   await writeFile(file.path, stringify(file.variables))
 }
+
+/**
+ * Given an .env file content, generates a new one with new values
+ * without removing already existing lines.
+ * @param envFileContent {string | null} .env file contents.
+ * @param updatedValues {[key: string]: string}} object containing new env variables values.
+ */
+export function patchEnvFile(envFileContent: string | null, updatedValues: {[key: string]: string}): string {
```

Comment:
Probably need to test all this with Windows `\r\n` style line breaks too (We should cope with them, and we should preserve them)

Reply:
@shauns does the last commit test what you were pointing out here correctly?